services/image-service.ts

```diff
@@ -0,0 +1,135 @@
+import { createTable } from "../helpers";
+
+//todo: plamen5kov: moved most of emulator-service here as a temporary solution untill after 3.0-RC
+export class ImageService implements Mobile.IImageService {
```

Comment:
hm, why is this called ImageService? This way I would think it works with images, for example in App_Resources dir.